src/etest_runner.erl

```diff
@@ -80,12 +90,19 @@ testfuns(Module) ->
     MakeApplicative = fun({FunName, _}) ->
         fun() ->
             inc(tests),
+
             Msg = lists:flatten(io_lib:format("~p:~p ", [Module, FunName])),
-            io:format(
-                string:left(Msg, 80, $.) ++ "\n" ++
-                string:left("",  80, $=) ++ "\n"
-            ),
```

Comment:
Wrap test fun in `...`, `===` to tell that the output received in between belongs to the test fun.